phpBB/docs/events.md

```diff
@@ -70,6 +70,18 @@ acp_ext_details_notice
 * Since: 3.1.11-RC1
 * Purpose: Add extension detail notices after version check information.
 
+acp_ext_list_available_name_after
+===
+* Location: adm/style/acp_ext_list.html
+* Since: 3.1.13-RC1
```

Comment:
I guess you meant 3.3.13-RC1? Should be changed to 3.3.14-RC1 now, since it won't make it into 3.3.13-RC1 anymore ;)